apps/documentation/components/packages/design/form/complexForm.tsx

```diff
@@ -37,6 +37,20 @@ export default function Form() {
         tooltip: 'Your email address',
       },
     },
+    {
```

Comment:
add this code component to field object below that displays the code people can copy to get the data.